client/my-sites/domains/domain-management/list/list-header.jsx

```diff
@@ -27,6 +28,24 @@ class ListHeader extends React.PureComponent {
 		const { headerClasses, translate } = this.props;
 
 		const listHeaderClasses = classNames( 'list-header', headerClasses );
+		const emailPopoverText = isEnabled( 'titan/phase-2' )
+			? translate(
+					'You can receive email using your custom domain by purchasing Email or %(googleMailService)s, or by ' +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **6** times:
`translate( 'You can receive email using your custom domain by using email forwarding or by purchasing Titan Mail or %(googleMailService)s. Note that email forwarding requires a plan subscription.' )` *ES Score: 10*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/rename-titan-to-email)